vispTypes.hpp

```diff
@@ -38,6 +39,23 @@ namespace visp
         float gain_at_infinity;
         float slope_at_zero;
     };
+
+    /** Group relevant information about the controller
+     *
+     *  error        - the norm-2 of the error vector
+     *  residual     - value expressed in meter that represents the
+                       "quality" of the measurement"
+     *  desired_pose - the current setpoint
+     *  current_pose - the current pose
+     */
+    struct controllerState
+    {
+        double error;
```

Comment:
After the reading, all is clear for me now! 
So, in that case the most correct time would be the time what the frame was captured by the camera, right?
As this information is not available for this component (~~because the input ports are not timestamped...shame on me~~) you suggested to take the last received visual sample, but the most correct to do here would be that `perception/orogen/apriltags` timestamp the output corners with the same frame timestamp, and then the velocities here would have the same timestamp, that would be the required velocities for that time in the real world. Right?

I timestamped the "controller_state" as the time it receive the corners, but I opened a [trello card](https://trello.com/c/ZS64Ddfu) to fix this mess. OK?